pkg/controller/machinehealthcheck/machinehealthcheck_controller_test.go

```diff
@@ -265,19 +284,193 @@ func TestReconcile(t *testing.T) {
 		}
 		result, err := r.Reconcile(request)
 		if result != tc.expected.result {
-			t.Errorf("Expected %v, got: %v", tc.expected.result, result)
+			t.Errorf("Test case : %v. Expected: %v, got: %v", tc.node.Name, tc.expected.result, result)
 		}
 		if tc.expected.error != (err != nil) {
-			t.Errorf("Expected error, got %v", err)
+			var errorExpectation string
+			if !tc.expected.error == true {
+				errorExpectation = "no"
+			}
+			t.Errorf("Test case: %s. Expected %s error, got: %v", tc.node.Name, errorExpectation, err)
 		}
 	}
 }
 
 // newFakeReconciler returns a new reconcile.Reconciler with a fake client
-func newFakeReconciler(initObjects ...runtime.Object) reconcile.Reconciler {
+func newFakeReconciler(initObjects ...runtime.Object) *ReconcileMachineHealthCheck {
 	fakeClient := fake.NewFakeClient(initObjects...)
 	return &ReconcileMachineHealthCheck{
 		client: fakeClient,
 		scheme: scheme.Scheme,
 	}
 }
+
+func TestHasMachineSetOwner(t *testing.T) {
```

Comment:
nit: wording: `TestMachineSetHasOwner`?

Reply:
Actually is right, is testing if "it has a machineSet owner"? happy to reword if you have a better suggestion

Reply:
ok